src/intra_edge.rs

```diff
@@ -179,14 +188,15 @@ impl EdgeBranch {
 impl<const SB128: bool, const N_BRANCH: usize, const N_TIP: usize>
     IntraEdge<SB128, N_BRANCH, N_TIP>
 {
-    fn init_mode_node(
-        &mut self,
+    #[must_use]
```

Comment:
Why the `#[must_use]` attribute here?

Reply:
It's not really necessary, but since I switched from `&mut Self` to `Self -> Self`, I wanted to make sure the return value wasn't forgotten about, since `this.init_mode_node()` needed to be replaced with `this = this.init_mode_node()`.